src/gitgraph.js

```diff
@@ -1088,7 +1085,7 @@
         var textHeight = _getFontHeight( this.tagFont );
         _drawTextBG( this.context,
           this.x - this.dotSize / 2,
-          ((this.parent.columnMax + 1) * this.template.commit.tag.spacingY) - this.template.commit.tag.spacingY / 2 + (this.parent.tagNum % 2) * textHeight * 1.5,
```

Comment:
This change was not meant to be committed. I have since reversed this changer and recommitted.